templates/aerie-alb.template.yaml

```diff
@@ -0,0 +1,120 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Service and Task definitions for the Aerie ECS Deployment
+
+Parameters:
+  ProjectName:
+    Type: String
+    Default: ast-aerie
+  VpcId:
+    Type: AWS::EC2::VPC::Id
+  PublicSubnetIds:
+    Type: List<AWS::EC2::Subnet::Id>
+  EnableAlbAccessLogs:
+    Type: String
+    Default: "false"
+    AllowedValues:
+      - "true"
+      - "false"
+
+Mappings:
+  ElbAccountIds:
+    us-east-1: {AccountId: "127311923021"}
+    us-east-2: {AccountId: "033677994240"}
+    us-west-1: {AccountId: "027434742980"}
+    us-west-2: {AccountId: "797873946194"}
+    af-south-1: {AccountId: "098369216593"}
+    ca-central-1: {AccountId: "985666609251"}
+    eu-central-1: {AccountId: "054676820928"}
+    eu-west-1: {AccountId: "156460612806"}
+    eu-west-2: {AccountId: "652711504416"}
+    eu-south-1: {AccountId: "635631232127"}
+    eu-west-3: {AccountId: "009996457667"}
+    eu-north-1: {AccountId: "897822967062"}
+    ap-east-1: {AccountId: "754344448648"}
+    ap-northeast-1: {AccountId: "582318560864"}
+    ap-northeast-2: {AccountId: "600734575887"}
+    ap-northeast-3: {AccountId: "383597477331"}
+    ap-southeast-1: {AccountId: "114774131450"}
+    ap-southeast-2: {AccountId: "783225319266"}
+    ap-southeast-3: {AccountId: "589379963580"}
+    ap-south-1: {AccountId: "718504428378"}
+    me-south-1: {AccountId: "076674570225"}
+    sa-east-1: {AccountId: "507241528517"}
+    us-gov-west-1: {AccountId: "048591011584"}
+    us-gov-east-1: {AccountId: "190560391635"}
+
+Conditions:
+  AlbAccessLogs: !Equals [!Ref EnableAlbAccessLogs, "true"]
+
+Resources:
+  AerieAlbLogsBucket:
+    Condition: AlbAccessLogs
+    Type: AWS::S3::Bucket
+
+  AlbBucketPolicy:
+    Condition: AlbAccessLogs
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref AerieAlbLogsBucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            AWS: !Sub
+              - "arn:${Partition}:iam::${ElbAccountId}:root"
+              - Partition: !Ref AWS::Partition
+                ElbAccountId: !FindInMap [ElbAccountIds, !Ref AWS::Region, AccountId]
+          Action: s3:PutObject
+          Resource: !Sub "arn:${AWS::Partition}:s3:::${AerieAlbLogsBucket}/ecs/aerie-alb/AWSLogs/${AWS::AccountId}/*"
```

Comment:
looks like you're putting ELB Access logs to specific accounts? Are these JPL Accounts? Not sure that's gonna fly.

Reply:
I know it looks strange, but this is not any sort of account ID that we own and does not affect the location of logs.
This access policy is required for the ELB service to move logs into the bucket as according to:
https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy